android/envoy/src/main/kotlin/org/greatfire/envoy/UrlUtil.kt

```diff
@@ -149,5 +149,63 @@ class UrlUtil {
 
             return sanitizedString
         }
+
+        @JvmStatic
+        fun getServiceFromUrl(url: String): String? {
```

Comment:
where is this method used?

Reply:
it's used in a few places in gf-droid code as well as in joinToSanitizedAndTruncatedString and sanitizeUrl which is used a few places